File: cfgs/java_cfgs_foo/CHECK_WHETHER_GIVEN_DEGREES_VERTICES_REPRESENT_GRAPH_TREE.java
Nodes
1;special;;;
2;statement;int deg_sum = 0;14;
3;statement;int i = 0;15;
4;conditional;foo_1 < foo_2;16;
5;statement;deg_sum += foo_1[foo_2];18;
6;statement;i++;17;
7;statement;return (2 * (foo_1 - 1)) == foo_2;20;
8;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;7;
4;6;
5;4;
6;8;
7;

File: cfgs/pythoncfgs/CHECK_WHETHER_GIVEN_DEGREES_VERTICES_REPRESENT_GRAPH_TREE.py
Nodes
1;statement;foo_1 = fun_1(foo_2);8;
2;conditional;2 * (foo_1 - 1) == foo_2;9;
3;statement;return false;12;
4;statement;return true;10;
5;exit;;
Edges
0;2;
1;3;4;
2;5;
3;5;
4;
